x-pack/plugins/enterprise_search/server/integrations.ts

```diff
@@ -817,6 +817,41 @@ export const registerEnterpriseSearchIntegrations = (
       shipper: 'enterprise_search',
       isBeta: false,
     });
+    customIntegrations.registerCustomIntegration({
+      id: 'confluence_data_center',
+      title: i18n.translate(
+        'xpack.enterpriseSearch.integrations.connectors.confluenceDataCenterTitle',
+        {
+          defaultMessage: 'Confluence Data Center',
+        }
+      ),
+      description: i18n.translate(
+        'xpack.enterpriseSearch.integrations.connectors.confluenceDataCenterDescription',
+        {
+          defaultMessage: 'Search over your content on Confluence Data Center.',
+        }
+      ),
+      categories: [
+        'enterprise_search',
+        'elastic_stack',
+        'custom',
+        'connector',
+        'connector_client',
+        'confluence_data_center',
```

Comment:
nitpick: Confluence data center shouldn't be in the categories list, these are used to filter things in integrations and there's no 'Confluence Data Center' filter.